File: 1021.c
#include<stdio.h>

int main(){
	
	double n;
	int nota100, nota50, nota20, nota10, nota5, nota2, moeda1, moeda050, moeda025, moeda010, moeda005, moeda001;
	
		scanf("%lf", &n);
		
			nota100 = n / 100;
			n -= nota100 * 100;
			nota50 = n / 50;
			n -= nota50 * 50;
			nota20 = n / 20;
			n -= nota20 * 20;
			nota10 = n / 10;
			n -= nota10 * 10;
			nota5 = n / 5;
			n -= nota5 * 5;
			nota2 = n / 2;
			n -= nota2 * 2;
			moeda1 = n;
			n -= moeda1 * 1;

			n = n * 100;
			moeda050 = n / 50;
			n -= moeda050 * 50;
			moeda025 = n / 25;
			n -= moeda025 * 25;
			moeda010 = n / 10;
			n -= moeda010 * 10;
			moeda005 = n / 5;
			n -= moeda005 * 5;
			moeda001 = n;
			
		
		
		printf("NOTAS:\n");
		printf("%d nota(s) de R$ 100.00\n", nota100);
		printf("%d nota(s) de R$ 50.00\n", nota50);
		printf("%d nota(s) de R$ 20.00\n", nota20);
		printf("%d nota(s) de R$ 10.00\n", nota10);
		printf("%d nota(s) de R$ 5.00\n", nota5);
		printf("%d nota(s) de R$ 2.00\n", nota2);
		
		printf("MOEDAS:\n");
		printf("%d moeda(s) de R$ 1.00\n", moeda1);
		printf("%d moeda(s) de R$ 0.50\n", moeda050);
		printf("%d moeda(s) de R$ 0.25\n", moeda025);
		printf("%d moeda(s) de R$ 0.10\n", moeda010);
		printf("%d moeda(s) de R$ 0.05\n", moeda005);
		printf("%d moeda(s) de R$ 0.01\n", moeda001);
	
		
		return 0;
		
}

File: Iniciante/1021.py
n = float(input())

print('NOTAS:')
aux = int(n // 100); print(f'{aux} nota(s) de R$ 100.00'); n = n % 100
aux = int(n // 50); print(f'{aux} nota(s) de R$ 50.00'); n = n % 50
aux = int(n // 20); print(f'{aux} nota(s) de R$ 20.00'); n = n % 20
aux = int(n // 10); print(f'{aux} nota(s) de R$ 10.00'); n = n % 10
aux = int(n // 5); print(f'{aux} nota(s) de R$ 5.00'); n = n % 5
aux = int(n // 2); print(f'{aux} nota(s) de R$ 2.00'); n = n % 2

print('MOEDAS:')
aux = int(n // 1); print(f'{aux} moeda(s) de R$ 1.00'); n = n % 1; n = n*100
aux = int(n // 50); print(f'{aux} moeda(s) de R$ 0.50'); n = n % 50
aux = int(n // 25); print(f'{aux} moeda(s) de R$ 0.25'); n = n % 25
aux = int(n // 10); print(f'{aux} moeda(s) de R$ 0.10'); n = n % 10
aux = int(n // 5); print(f'{aux} moeda(s) de R$ 0.05'); n = int(n % 5)
print(f'{n} moeda(s) de R$ 0.01')